### DISCUSSION TITLE
Security scoped benchmark could not be resolved

### DISCUSSION THREAD

**Discussion by tomwolsky** [latenitefilms/BRAWToolbox#45] [Support] [answered]
Open my BRAW library this morning and one of the files shows as missing. It has an error of security scoped benchmark could not be resolved. The file was there yesterday as the cache shows an image briefly in the filmstrip. The file has not moved and is still in the BRAW folder on the system. Relink doesn't work as it doesn't put up any missing file and won't show anything with All either. Reveal in Finder is grayed out. I've never seen anything quite like this. I was able to reimport the file without problem. 
<img width="279" alt="60%_Screenshot 2023-01-26 at 06 56 06" src="https://user-images.githubusercontent.com/18217685/214830785-014ff5fd-b6fc-47c2-8766-dc8880b60fbc.png">

**Comment by latenitefilms** [comment, marked as answer]
Maybe related to trying to use the uninstall tool? If you uninstall you break the bookmarks.

As a workaround you could try just granting BRAW Toolbox access to your drive via this:

https://brawtoolbox.io/collaboration/

**Reply by latenitefilms** [reply]
There's also a relink tool in the toolbox:

https://brawtoolbox.io/toolbox/#relink-braw-clips-within-an-event

**Reply by tomwolsky** [reply]
Thanks Chris. Odd that it was only on one shot all of a sudden. I’ll  uninstall again to see if it reproduces.

**Reply by latenitefilms** [reply]
Another thing you can do if it's just a single clip is just manually relink via the BRAW Toolbox effect. Just open the same clip again from the FCPX Inspector.

Are you able to email me your logs files to see if I can work out what went wrong?

https://brawtoolbox.io/troubleshooting/#ive-run-into-a-bug-where-can-i-find-the-log-files

You can upload here:

https://latenitefilms.digitalpigeon.com

**Comment by tomwolsky** [comment]
I uploaded the files in the Application Support folder.  A zip file called ToolboxLogs. 

All the best,

Tom

**Reply by latenitefilms** [reply]
Huh, weirdly, I don't see any errors in your FxPlug4 logs. It looks like it might have been Final Cut Pro freaking out, rather than BRAW Toolbox?
